src/shared/conf-parser.c

```diff
@@ -602,7 +603,11 @@ int config_parse_many(
         if (r < 0)
                 return r;
 
-        return config_parse_many_files(conf_files, files, sections, lookup, table, flags, userdata, ret_stats_by_path);
+        r = config_parse_many_files(conf_files, files, sections, lookup, table, flags, userdata, ret_stats_by_path);
+
+        if (ret_dropin_files) *ret_dropin_files = TAKE_PTR(files);
```

Comment:
We never put the conditional and statement on the same line, so please add a newline here after the conditional.